tests/cli/1PasswordImportTests.js

```diff
@@ -0,0 +1,54 @@
+"use strict";
+
+const path = require("path");
+
+const ButtercupImporter = require("../../source/index.js");
+const { Archive, Group, Entry } = require("buttercup");
+
+module.exports = {
+
+    setUp: function(cb) {
+		this.exampleArchive = path.resolve(__dirname, "../resources/test-1password.1pif");
+		cb();
+	},
+
+    export: {
+
+        createsArchive: function(test) {
+			ButtercupImporter
+				.importFrom1PIF(this.exampleArchive)
+				.then(function(archive) {
+					test.ok(archive instanceof Archive, "Archive should be a Buttercup archive instance");
+					test.done();
+				});
+		},
+
+        containsGroups: function(test) {
+			ButtercupImporter
+				.importFrom1PIF(this.exampleArchive)
+				.then(function(archive) {
+					const rootGroup = archive.getGroups()[0];
+					test.ok(/Imported 1password archive/i.test(rootGroup.getTitle()), "Root group should be named correctly");
+					const generalGroup = rootGroup.findGroupsByTitle("General")[0];
+                    test.ok(generalGroup instanceof Group, "General group should exist");
+                    const subGroup = generalGroup.findGroupsByTitle("Sub")[0];
+                    test.ok(subGroup instanceof Group, "Sub group should exist");
+				})
+                .then(test.done);
+		},
+
+        containsEntries: function(test) {
```

Comment:
and here too. I think if you cut the lines and paste them again vscode will fix it all

![image](https://cloud.githubusercontent.com/assets/768052/25721195/4175f364-3118-11e7-9d7a-89af7ee89a28.png)